user/build-matrix.md

```diff
@@ -0,0 +1,294 @@
+---
+title: Build Matrix
+layout: en
+---
+
+The build matrix is a way to run builds on multiple versions of language runtimes, on multiple operating systems, and with multiple versions of environment variables.
```

Comment:
> multiple versions of environment variables.

doesn't seem right to me. Perhaps "different values of environment variables". The matrix expansion over `env` is not necessarily tied to a single environment variable, either, so I'm not sure how best to say it.

```yaml
env:
  - VAR1=val1
  - VAR1=val2
  - VAR2=val2
```

Reply:
:100: